app/Http/Controllers/Api/v1/Campaign/UserApiController.php

```diff
@@ -26,7 +27,19 @@ public function index(Campaign $campaign)
         $this->authorize('access', $campaign);
         $this->authorize('update', $campaign);
 
-        return UserResource::collection($campaign->users()->paginate());
+        return UserResource::collection($campaign->users()
+        ->lastSync(request()->get('lastSync'))
+        ->paginate());
+    }
+
+    public function user(Campaign $campaign, User $user)
```

Comment:
To keep with Laravel's naming convention, please use `show()` as the function name :)